StreamChatUITestsAppUITests/MockServer/MessageResponses.swift

```diff
@@ -323,13 +323,15 @@ extension StreamMockServer {
         let user = setUpUser(source: message, details: UserDetails.lukeSkywalker)
         let mockedMessage = mockDeletedMessage(message, user: user)
         
-        websocketMessage(
-            channelId: channelId,
-            messageId: messageId,
-            timestamp: timestamp,
-            eventType: .messageDeleted,
-            user: user
-        )
+        websocketDelay {
```

Comment:
@alteral please `weakify` self in all places where `websocketDelay` is called.

Reply:
Done 👍